### DISCUSSION TITLE
2022-07-29-pd-code

### DISCUSSION THREAD

**Discussion by giscus** [tw93/tw93.github.io#128] [General]
# 只想靠两手，向理想挥手 - Tw93

Hi，我是 Tw93，是一个前端工程师，会一点 Swift / Rust / Python，喜欢折腾各种新技术。Share只想靠两手，向理想挥手，这是来自 Beyond 的《不再犹豫》里面的一句歌词。大概从初中起就喜欢听 Beyond 的歌，陆陆续续一直听到大学，每一次看他们以前演唱会视频都特别兴奋和激动。带给我的大部分是积极向上的力量和一种酷酷的精神。回到文章，这篇文章主要是记录自己找工作的过程，顺便总结一下大学四年的校园生活 ...

https://tw93.fun/2015-05-18/About-My-College-And-Alibaba.html

**Comment by Sure-Will** [comment]
我是河南大学今年毕业的计算机本科生，看完前辈的博客很有感想，想起来自己当初也是怀揣着计算机的梦想选择了这个专业，然而本科期间喜欢打电子游戏荒废了学业，四年大学时光图书馆只去了一趟，挂过一门科，感觉自己事事都爱琢磨点，但又不爱扣太仔细，代码什么的都是略知皮毛，从来没有想过精进自己的技术，一战考研失败后觉得要找一个班上，刚开始找工作有一个星期就被家里人叫停了，还是想让我考研，如今在家二战考研，由于八月底报考院校突然改考，目前对于新学的科目也并不是很有信心，个人平常闲暇时喜欢在github上翻来覆去的了解一些有趣的项目，偶然翻到了前辈的这个pake项目，打开主页翻看前辈的博客，看到前辈不是科班出身，却比我更知道要在大学里去追寻什么。我本人在大学期间跟身边的朋友关系都很好，但在学业上几乎没有什么交流，一起玩耍倒是很积极，可能这就是学校之间的学习氛围差距吗，不过归根结底还是我自己的原因很大。现如今回首自己的四年大学生涯，没能好好珍惜四年时光真的好可惜啊，当然也怪我自己，我是一个非常没有上进心的人，也可能是在学习方面吧。说来惭愧在学生会上我倒很是积极一直干了三年干到了正主席，然而我也并没有选择行政保研这条路，总是想考一个名牌大学，可能我的性格并不适合在大学里了吧，但我还是很想考入一所名牌大学。看完前辈本科四年的历程，以及找工作、跟HR交流面试、对自己个人的规划等等等等，内心很是感慨，写了很多有些不成逻辑，最后祝前辈工作顺心，万事如意，也希望自己能够振作起来，能够在如今的时代洪流中找到自己的路。

**Reply by Char2sGu** [reply]
祝你工作如意生活顺利！
